Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/InvoluntaryProject/SearchSchool.cshtml.cs

```diff
@@ -66,7 +66,10 @@ public IActionResult OnPost(string ukprn)
 			var splitSearch = SplitOnBrackets(SearchQuery);
 			if (splitSearch.Length < 2) return Page();
 
-			return RedirectToPage(Links.InvoluntaryProject.SearchTrusts.Page, new { urn = splitSearch[1], ukprn });
+			if (string.IsNullOrWhiteSpace(r)) return RedirectToPage(Links.InvoluntaryProject.SearchTrusts.Page, new { urn = splitSearch[1], ukprn });
+
+			Response.Redirect(r);
```

Comment:
Could we use a more descriptive variable rather than "r", other than that happy to approve.

Reply:
This has now been reworked slightly. Noticed a bug with it.